README.md

```diff
@@ -1,4 +1,22 @@
-# susi_hardware
+# Susi Hardware
 
 [![Join the chat at https://gitter.im/fossasia/susi_hardware](https://badges.gitter.im/fossasia/susi_hardware.svg)](https://gitter.im/fossasia/susi_hardware?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 Hardware for SUSI AI
```

Comment:
Move this to next line. The badge and "Hardware for SUSI AI" are in same line. Looking awkward.